shaman/controllers/repos/projects.py

```diff
@@ -16,6 +16,12 @@ def __init__(self, project_name):
     def index(self):
         abort(405)
 
+    @index.when(method='GET', template='json')
+    def index_get(self):
+        if not self.project:
+            abort(404)
+        return self.project
```

Comment:
Shouldn't this endpoint return information about what types of repos the project has?